documentation/src/main/asciidoc/userguide/chapters/statistics/Statistics.adoc

```diff
@@ -0,0 +1,159 @@
+[[statistics]]
+== Statistics
+:stat-sourcedir: ../../../../../../../hibernate-core/src/main/java/org/hibernate/stat
+
+Hibernate can gather all sorts of statistics which can help you get a better insight into what Hibernate does behind the scenes.
+
+By default, the statistics are not collected because this incurs an additional processing and memory overhead. To instruct Hibernate to start collecting statistics, you need to set the `hibernate.generate_statistics` configuration property to `true`:
+
+====
+[source,xml]
+----
+<property
+    name="hibernate.generate_statistics"
+    value="true"
+/>
+----
+====
+
+[[statistics-methods]]
+=== org.hibernate.stat.Statistics methods
```

Comment:
Maybe backticks around the class name? I don't know if you do it for titles in the ORM documentation.

Reply:
For titles, adding backticks makes the title look bad because of mixing two different fonts.